### ISSUE TITLE
Eosinophils nuclear hypersegmentation

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7461]
**Preferred term label:**
Eosinophils nuclear hypersegmentation

**Synonyms**
Nuclear hypersegmentation of eosinophils

**Definition (free text, please give PubMed ID)**
An excessive division of the lobes of the nucleus of eosinophils. Eosinophils with more than two nuclear lobes are commonly regarded as hypersegmented, because seventy to ninety percent of peripheral blood eosinophils have two nuclear lobes.
PMID:10732836 
PMID:25313414

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
PMID: 7809065  1/1 Fig 2C

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
